examples/crowd-funding/README.md

```diff
@@ -0,0 +1,237 @@
+<!-- cargo-rdme start -->
+
+# Crowd-funding Example Application
+
+This example application implements crowd-funding campaigns using fungible tokens in
+the `fungible` application. This demonstrates how to compose applications together and
+how to instantiate applications where one chain has a special role.
+
+Once this application is built and its bytecode published on a Linera chain, the
+published bytecode can be used to create different instances, where each
+instance represents a different campaign.
```

Comment:
The instantiation on one chain and then putting it on other chains is a common feature to all applications I believe. I think it is important to emphasize what is different and what is common for crowd-funding.